Create membersandsettings.png (#11106)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/ediscovery-teams-workflow.md

For more information about the increased case limits, see [Use the new case format in eDiscovery (Premium)](ediscovery-new-case-format.md).

1. Go to <https://compliance.microsoft.com> and sign in.

2. In the left navigation pane of the Microsoft Purview compliance portal, click **eDiscovery > Premium**.

3. On the **eDiscovery (Premium)** page, click the **Cases** tab, and then click **Create a case**.

>    The **New eDiscovery case** flyout page is displayed. The **Case format** section provides the option to create a case using the new case format, Click **Next** to proceed to the **Members and settings** Page.

Reviewer: ```suggestion
   The **New eDiscovery case** flyout page is displayed. The **Case format** section provides the option to create a case using the new case format. Select **Next** to proceed to the **Members and settings** page.
```

   ![Large case option on the New eDiscovery case page.](..\media\AeDNewCaseFormat1.png)
   

>  4. On the **Members and Settings** Page, Add team members and configure settings and Click on **next** to proceed to the **Summary** page

Reviewer: ```suggestion
 4. On the **Members and Settings** page, add team members and configure settings, then select **next** to proceed to the **Summary** page.
```

   ![Members and settings page](..\media\membersandsettings.png

> 4. On the Summary of the Case, Review the settings and click **Submit** to create the case.

Reviewer: ```suggestion
4. In the summary of the case, review the settings and select **Submit** to create the case.
```

8. On the **Conditions** wizard page, configure the search query to collect Teams content from the data sources that you specified on the previous wizard pages. You can use various keywords and search conditions to narrow the scope of the collection. For more information, see [Build search queries for collections](ediscovery-building-search-queries.md).

>    To help ensure the most comprehensive collection of Teams chat conversations (including 1:1, group, and channel chats) use the **Message Kind** condition and select **Equals Any of** option and type **Microsoftteams**. We also recommend including a date range or several keywords to narrow the scope of the collection to items relevant to your investigation. Here's a screenshot of a sample query using the **Type** and **Date** options:

Reviewer: ```suggestion
   To help ensure the most comprehensive collection of Teams chat conversations (including 1:1, group, and channel chats), use the **Message Kind** condition, select the **Equals any of** option, and enter **Microsoftteams**. We also recommend including a date range or several keywords to narrow the scope of the collection to items relevant to your investigation. Here's a screenshot of a sample query using the **Type** and **Date** options:
```
